pkg/operator/starter.go

```diff
@@ -604,12 +604,25 @@ func prepareOauthAPIServerOperator(ctx context.Context, controllerContext *contr
 		return err
 	}
 
+	operatorConfig, err := operatorCtx.configClient.ConfigV1().APIServers().Get(ctx, "cluster", metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	// TODO(4.7): switch over to secure access-token logging by default and delete old non-sha256 tokens
```

Comment:
who is going to delete old non-sha256 tokens?

Reply:
We will block upgrade. To be done.